logos/benchmarks/benchmark_logos.py

```
@@ -3458,107 +3461,124 @@ class ModelStateSnapshot:
+    return f"https://{host}:{admin_port}"
+
+
+async def _discover_provider_names(logos_url: str, logos_key: str) -> "dict[int, str]":
+    """Best-effort {provider_id: friendly_name} from the (non-root) vram-stats
+    endpoint, so the live lane poller works (and charts keep human names like
+    deimama/deipapa) even when --calibration-provider-ids was not given."""
+    url = f"{logos_url.rstrip('/')}/logosdb/get_ollama_vram_stats"
+    headers = {"logos_key": logos_key, "Content-Type": "application/json"}
+    names: "dict[int, str]" = {}
+    try:
+        async with httpx.AsyncClient(verify=False, timeout=httpx.Timeout(10.0)) as client:
```

Ruff (0.15.18)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)